meson.build

```diff
@@ -113,6 +112,12 @@ warning_flags = [
 cc = meson.get_compiler('c')
 add_project_arguments(cc.get_supported_arguments(warning_flags), language : 'c')
 
+# use honggfuzz for parallel persistent fuzzing
+honggfuzz = find_program('honggfuzz', required: cc.has_function('HF_ITER'))
+if honggfuzz.found()
+  conf.set('HAVE_HF_ITER', '1')
+endif
+
```

Comment:
you know I just realized something about doing it this way though.  If `honggfuzz` is _installed_ that will turn on fuzzing everywhere.  is that expected?  Or does this need an extra meson option?

Reply:
>  is that expected 

Yes, you can use honggfuzz without compiling with the custom clang compiler, but you can't use the persistent mode.

Reply:
OK.